index.html

```diff
@@ -0,0 +1,111 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microverse Project</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;1,200&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+    <div class="container">
+        <h1 class="text-center" id="title">Microverse Survey Form</h1>
+        <p class="text-center" id="description">
+            Thank you for taking the time to help us improve the platform
+        </p>
+        <div class="col-md-6 mx-auto form p-4" >
+
+        <form  id="survey-form" class="col-md-12  ">
+            <div class="form-group my-4">
+                <label class="mb-3" id="name-label" for="name">Name</label>
+                <input class="form-control" required type="text" id="name" placeholder="Enter your name" />
+            </div>
+            <div class="form-group my-4">
+                <label class="mb-3" id="email-label" for="email">Email</label>
+                <input class="form-control" required type="email" id="email" placeholder="Enter your email" />
+            </div>
+            <div class="form-group my-4">
+                <label class="mb-3" id="number-label" for="number">Age (optional)</label>
+                <input class="form-control" min="5" max="100" id="number" type="number" pattern="[0-9]"
+                    placeholder="Age" />
+            </div>
```

Comment:
- [ ] Attribute “pattern” is only allowed when the input type is “email”, “password”, “search”, “tel”, “text”, or “url”. Please remove the pattern attribute or change the input type to the appropriate type